lib/kino/input.ex

```diff
@@ -384,17 +384,17 @@ defmodule Kino.Input do
 
     if min && max && Date.compare(min, max) == :gt do
       raise ArgumentError,
-            "expected :min to be less than :max, got: #{inspect(min)} and #{inspect(max)}"
+            ":min = #{inspect(min)} after :max = #{inspect(max)}"
```

Comment:
It is a common phrasing in Elixir, but it's fine to reword, we should be more descriptive though:

```suggestion
            "empty date range, :min (#{inspect(min)}) is after :max (#{inspect(max)})"
```